src/test/java/testsuite/BaseTestCase.java

```diff
@@ -115,8 +115,7 @@
     private List<String[]> createdObjects;
 
     /** The driver to use */
-    protected String dbClass = com.mysql.cj.jdbc.Driver.class.getName();
```

Comment:
Hmm why?

Reply:
since this class establishes the initial connection that will be used in a bunch of the tests, it first does dbClass.forName to register the driver, then establishes the test connection with DriverManager.getConnection. If I don't change it to our new driver, the test connection can't be established because the original driver can't be registered anymore, and a bunch of tests fail